docs/deploy.md

```diff
@@ -134,18 +146,55 @@ For the official homeworld servers:
                 GSSAPIAuthentication yes
                 GSSAPIKeyExchange no
                 GSSAPIDelegateCredentials no
+                PreferredAuthentications gssapi-with-mic
+                LocalForward 5901 localhost:5901
+                LocalForward 5902 localhost:5902
+                LocalForward 5903 localhost:5903
+                LocalForward 5904 localhost:5904
+                LocalForward 5905 localhost:5905
+                LocalForward 5906 localhost:5906
+                LocalForward 5910 localhost:5910
+
+        # Note that you will need Kerberos tickets
+        # (generate them with kinit)
+        # to access the development server.
     $ scp preseeded.iso toast:/srv/preseeded.iso
 
-## Set up the supervisor operating system
+# Setting up the machines
+
+## For development only: Rebuilding the virtual machines
+
+For development, we're using a set of virtual machines on toast. To simulate cluster bringup, we destroy all the virtual machines and rebuild them using a script on toastfs-dev. On toastfs-dev (access with ``ssh toast``):
+
+    $ ~/hyades/rebuild-homeworld-cluster.sh /srv/preseeded.iso
```

Comment:
One convention (optional here) is that, when a command is run as root, you use a `#` instead of a `$`. But that's not very important.